packages/@okta/vuepress-site/docs/guides/oin-lifecycle-mgmt-overview/index.md

```diff
@@ -0,0 +1,115 @@
+
+After approval, the OktReward app is published as a lifecycle management integration (in addition to an SSO integration) in the OIN.
+
+### Example of customer admin integration journey with Okta
+
+Acme is a company that is using Okta to manage identity for their workforce. Acme is considering adding OktReward into their HR suite. Since OktReward is in the OIN with SSO and lifecycle management use cases, Acme knows that they can integrate OktReward securely into their existing Okta-managed SSO and automated user lifecycle flow with minimal effort. In addition, Acme can access OktRewards audits for their compliance needs.
+
+Alicia is an admin at Acme and performs the following tasks to integrate OktReward:
+
+* Adds the OktReward app integration in the OIN catalog from the Okta Admin Console.
+
+* Follows the instructions provided by OktReward to configure the SSO and Okta Workflows integration.
+
+* Configures the Okta authentication policy and the groups of Acme employees with various access levels to the OktReward app.
+
+* Imports user profiles from Okta to the OktReward app.
+
+* Tests signing in to the OktReward app with existing Okta credentials to verify the authentication flow.
+
+* Tests user onboarding, role changes, offboarding with HR to ensure that the user provisioning flow is working accurately.
+
+Acme's employees can sign in to the OktReward app with their existing Okta credentials and no additional OktReward app registration is required. When new employees join Acme, their HR staff adds user profiles in one Okta source. The user profiles are automatically provisioned to the apps with lifecycle management support, which now includes OktReward.
+
+### Example of a workforce lifecycle journey with Okta
+
+Key terms for lifecycle management in this workforce journey:
+
+* **Joiner**: Starting employment at an organization
+* **Mover**: Changing roles or teams within an organization
+* **Leaver**: Ending employment at an organization
+
+#### Joiner
+
+1. Lucy accepts an offer to join the Customer Support team at Acme Corp. The Acme HR team sets her start date in the employee directory and adds her to the “Customer Support” Okta group.
+1. On her start date, Okta automatically creates accounts for Lucy in all the tools used by every employee at Acme (for example: chat, corporate directory, intranet, and so on).
+1. Since Lucy is in the Customer Support Okta group, an account is also provisioned for Lucy in the tools used exclusively by the Customer Support team (for example, the case management tool).
+1. Acme uses Okta Workflows to provide custom onboarding actions, extending the usual account provisioning flows. Once Lucy’s account is activated, an onboarding flow is triggered based on Lucy's group. Since she is part of the Customer Support group, Lucy is added to a special folder in Acme’s cloud file storage service that the Customer Support team uses to share template files.
```

Comment:
```suggestion
1. Acme uses Okta Workflows to provide custom onboarding actions, extending the usual account provisioning flows. After Lucy’s account is activated, an onboarding flow is triggered based on Lucy's group. Since she is part of the Customer Support group, Lucy is added to a special folder in Acme’s cloud file storage service that the Customer Support team uses to share template files.
```